e2e/client/playwright/workspace.spec.ts

```diff
@@ -0,0 +1,156 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test.describe('Custom Workspace', async () => {
+    test.describe('Create custom workspace', async () => {
+        test('save custom workspace', async ({page}) => {
+            await restoreDatabaseSnapshot();
+            await page.goto('/#/workspace/monitoring');
+
+            await page.locator(s('monitoring--selected-desk')).click();
+            await page.locator(s('create-workspace')).click();
+            await expect(page.locator(s('workspace-modal', 'workspace-name'))).toBeVisible();
+            await expect(page.locator('button', {hasText: 'save'})).toBeDisabled();
+            await page.locator(s('workspace-modal')).locator(s('workspace-name')).fill('new custom workspace');
```

Comment:
go over all your tests and replace `page.locator(s('a')).locator(s('b'))`
with `page.locator(s('a', 'b'))`